### PR TITLE
DOCSP-41115: get started

### PR DESCRIPTION
**Pull request by rustagir**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-kotlin-sync/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-41115>
Staging - https://preview-mongodbrustagir.gatsbyjs.io/kotlin-sync/DOCSP-41115-get-started/get-started/

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/get-started/download-and-install.txt [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      {+language+} and JVM development, see `Get started with Kotlin/JVM
+      <{+kotlin-docs+}/docs/jvm-get-started.html>`__ in the {+language+}
+      language documentation.
+
+   .. step:: Create a {+language+} Project
+
+      This guide shows you how to add the {+driver-short+} dependencies
+      by using Gradle or Maven. We recommend that you use an integrated development
+      environment (IDE) such as IntelliJ IDEA or Eclipse IDE to configure
+      Gradle or Maven to build and run your project.
```
Comment (mcmorisi) [inline source/get-started/download-and-install.txt:23]: The instructions in step 1 dictate using IntelliJ to create a Kotlin project, so a reader who gets to step 2: Create a Project might have already created the project.

Suggestion would be maybe to reconfigure this to be installing the IDE in step 1 and linking to the Kotlin docs to create the project in step 2? Happy to discuss further.

File: source/get-started/connect-to-mongodb.txt [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+      the ``<connection URI string>`` placeholder with your MongoDB
+      Atlas connection string that you saved in the preceding step.
+
+      .. literalinclude:: /includes/get-started/DemoDataClassExample.kt
+         :language: kotlin
+         :caption: DemoDataClassExample.kt
+
+      .. note::
+      
+         This example uses a {+language+} data class to model MongoDB data.
```
Comment (mcmorisi) [inline source/get-started/connect-to-mongodb.txt:24]: I'm seeing that we have a task to add a Specialized Data Formats > Data Classes page, could be a good idea to leave a TODO here to link to that page for more info once it's built.

File: source/get-started/connect-to-mongodb.txt [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+         Movie(title=Before Sunrise, year=1995, directors=[Richard Linklater])
+
+       If you don't see any output or receive an error, check whether you
+       included the proper connection string in your application. Also, confirm
+       that you successfully loaded the sample dataset into your MongoDB Atlas cluster.
+
+       After completing this step, you have a working application that uses
+       the {+driver-short+} to connect to your MongoDB cluster, run a query on the
+       sample data, and print out the result.
```
Comment (mcmorisi) [inline source/get-started/connect-to-mongodb.txt:43]: Indentation here is breaking formatting

**Review by mcmorisi — CHANGES_REQUESTED** [review]
Looks great! One suggestion and a couple of nits.

File: source/get-started/download-and-install.txt (hunk 1 above)
Comment (rustagir) [reply source/get-started/download-and-install.txt:23]: How about we get rid of step 1 and move that information into step 2?

Comment (mcmorisi) [reply source/get-started/download-and-install.txt]: That also works!

File: source/get-started/download-and-install.txt [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+   .. step:: Create a {+language+} Project
+
+      First, make sure that your system has {+language+} installed and
+      running on JDK 1.8 or later.
+
+      This guide shows you how to create a project and add the
+      {+driver-short+} dependencies by using Gradle or Maven. We
+      recommend that you use an integrated development
+      environment (IDE) such as IntelliJ IDEA or Eclipse IDE to
+      configure Gradle or Maven to build and run your project.
```
Comment (mcmorisi) [inline source/get-started/download-and-install.txt:19]: I'd still lead with the "This guide..." sentence, maybe pull it up to the top level before the steps:
```suggestion

This guide shows you how to create a project and add the
{+driver-short+} dependencies by using Gradle or Maven. 

.. procedure::
   :style: connected

   .. step:: Create a {+language+} Project

      First, make sure that your system has {+language+} installed and
      running on JDK 1.8 or later.

      We recommend that you use an integrated development
      environment (IDE) such as IntelliJ IDEA or Eclipse IDE to
      configure Gradle or Maven to build and run your project.
```

**Review by mcmorisi — APPROVED** [review]
LGTM with one last comment!

**Review by rozza — APPROVED** [review]
LGTM!

**Pull request merged**